extension/js/common/core/mime.ts

```diff
@@ -15,7 +15,7 @@ const MimeBuilder = requireMimeBuilder();  // tslint:disable-line:variable-name
 const Iso88592 = requireIso88592();  // tslint:disable-line:variable-name
 
 type MimeContentHeader = string | { address: string; name: string; }[];
-type MimeContent = {
+export type MimeContent = {
```

Comment:
It's not clear that the export is needed, at least based on the code above. It seems it would work without the export.

Nothing wrong with exporting, but if it's not used elsewhere, better keep it local, so that it's easier to discover when it's unused (local unused variable goes gray, exported does not)

Reply:
oh, I forget to remove it,
I tried to use `Mime.decode` before but I didn' work well and I removed it, and forget to remove `export`.
Thanks